website/docs/r/cloudant.html.markdown

```diff
@@ -0,0 +1,131 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_cloudant"
+description: |-
+  Manages Cloudant instance.
+subcategory: "Cloud Databases"
+---
+
+# ibm_cloudant
+
+Provides a resource for IBM Cloudant. This allows an IBM Cloudant service instance to be created, updated, or deleted.
+For more information, about how to use this service, see the official [Cloudant documentation](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-getting-started-with-cloudant).
+
+## Example Usage
+
+```hcl
+resource "ibm_cloudant" "cloudant" {
+  name     = "cloudant-service-name"
+  location = "us-south"
+  plan     = "standard"
+
+  legacy_credentials  = true
+  include_data_events = false
+  capacity            = 1
+  enable_cors         = true
+
+  cors_config {
+    allow_credentials = false
+    origins           = ["https://example.com"]
+  }
+
+  timeouts {
+    create = "15m"
+    update = "15m"
+    delete = "15m"
+  }
+}
+```
+
+## Timeouts
+
+ibm_cloudant provides the following [Timeouts](https://www.terraform.io/docs/language/resources/syntax.html#operation-timeouts)
+configuration options:
+
+* `create` - (Default 10 minutes) The creation of the IBM Cloudant instance is considered failed if no response received.
+* `delete` - (Default 10 minutes) The update of the IBM Cloudant instance is considered failed if no response received.
+* `update` - (Default 10 minutes) The deletion of the IBM Cloudant instance is considered failed if no response received.
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `capacity` - (Optional, Number) A number of blocks of throughput units. For more details please read about [`blocks`](https://cloud.ibm.com/apidocs/cloudant#putcapacitythroughputconfiguration) parameter.
+  * Constraints: The default value is `1`. Capacity modification is not supported for `lite` plan.
+* `cors_config` - (Optional, Block List) Configuration for CORS. (see [below for nested arguments](#nestedblock--cors_config))
+  * Constraints: The minimum length is `1` item. Can conflict with `enable_cors` if it is set to `false`. In this case the CORS customization is not allowed.
+  * `allow_credentials` - (Optional, Boolean) Boolean value to allow authentication credentials. If set to true, browser requests must be done by using withCredentials = true.
```

Comment:
If set to true, browser requests must be done by using withCredentials = true.  could be as

If set to **true**, browser requests must be done by using `withCredentials = true`.